scripts/cxg_inspect.py

```diff
@@ -21,18 +21,20 @@
 
 
```

Comment:
Are all of the changes in this file linting related? Could you make a separate pr with the linting changes that are unrelated to the changes to fbs?